drivers/sms_ser.c

```diff
@@ -152,25 +152,25 @@ void sms_parse_results(uint8_t *rawvalues, SmsData *results) {
     results->lowbattery = ((byte & (mask << 6)) != 0) ? true : false;
     results->onbattery = ((byte & (mask << 7)) != 0) ? true : false;
 
-    upslogx(LOG_INFO, "beepon: %d", results->beepon);
-    upslogx(LOG_INFO, "shutdown: %d", results->shutdown);
-    upslogx(LOG_INFO, "test: %d", results->test);
-    upslogx(LOG_INFO, "upsok: %d", results->upsok);
-    upslogx(LOG_INFO, "bypass: %d", results->bypass);
-    upslogx(LOG_INFO, "lowbattery: %d", results->lowbattery);
-    upslogx(LOG_INFO, "onbattery: %d", results->onbattery);
+    upsdebugx(LOG_DEBUG, "beepon: %d", results->beepon);
+    upsdebugx(LOG_DEBUG, "shutdown: %d", results->shutdown);
```

Comment:
`upsdebugx()` does not deal with symbolic names (maybe should have, but...), but rather with relatively arbitrarily assigned verbosity level numbers (with debug setting "X or greater" the message would be seen), with nuances like this probably suitable for level 3 or more.

Reply:
So i will return to upslogx with LOG_DEBUG ? or you think is not necessary to show this ? Maybe a command can retrieve this.

Reply:
@jimklimov i cant signed my commits because has a lot of merges from master. Doing a `git rebase --signoff` its almost impossible to edit all commits, to sign it.

There is some tip for this ?

Reply:
oh.. i see, higher level to print, not debug.. 

i will see the script..